packages/extra-shot-mocha/tests/it.test.ts

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { describe } from "mocha";
+import { expect } from "chai";
+import { it } from "../src/it";
+
+describe("Advanced it tests", async () => {
+  it("should run with only title");
+
+  it("should run as normal mocha.it with sync function", function () {
+    expect(1).equals(1);
+  });
+
+  it("should run as normal mocha.it with async function", async function () {
+    expect(1).equals(1);
+  });
+
+  it("should run as normal mocha.it with sync arrow function", () => {
+    expect(1).equals(1);
+  });
+
+  it("should run as normal mocha.it with async arrow function", async () => {
+    expect(1).equals(1);
+  });
+
+  it("should inject ctx with async function", { a: 1 }, async function () {
+    expect(1).equals(1);
```

Comment:
Do we actually know whether the ctx is injected here?

Reply:
it relies on the reporter. Currently, only mochawesome can output the context. See https://www.npmjs.com/package/mochawesome

Reply:
<img width="699" alt="image" src="https://user-images.githubusercontent.com/71317774/153352038-c373da23-3948-4622-991e-edf810c7650d.png">

Reply:
Maybe check the injection when available.